addon/-private/execution_context/helpers.js

```diff
@@ -17,7 +17,7 @@ import $ from '-jquery';
  * @param {string} options.pageObjectKey   Key of method on PageObject which, when invoked, resulted in this call to `fillElement`
  * @return
  *
- * @throws Will throw an error if called on a contenteditable element that has `contenteditable="false"`
+ * @throws Will throw an error if called on a contenteditable element that has `contenteditable='false"`
```

Comment:
something went wrong here 😃

Reply:
Oops, good catch. Must've been search and replace